subsys/bluetooth/mesh/shell/shell_prop_cli.c

```diff
@@ -27,7 +27,13 @@ static void props_print(const struct shell *shell, int err, struct bt_mesh_prop_
 
 static int cmd_prop_client_props_get(const struct shell *shell, size_t argc, char *argv[])
 {
-	uint16_t id = (uint16_t)strtol(argv[1], NULL, 0);
+	int err = 0;
+	uint16_t id = shell_strtoul(argv[1], 10, &err);
```

Comment:
Forgot one :) Property ids also need hex support

Reply:
I don't really know why I haven't just provided hex support to all args by default. I will go through them and add it to most of them.